README.md

```diff
@@ -57,6 +57,21 @@ where
  * `cleanup_interval` (default: 1 hour) is the number of seconds between database cleanups 
  * `metrics_lifetime` (default: 24 hours) is the number of seconds that must pass from creation before each metric can be deleted during cleanup
 
+## Metrics visualisation with Grafana
+
+Metrics stored in the database using `membrane_rtc_engine_timescaledb` can be simply visualized using Grafana.
+To start the dashboard with RTC Engine metrics, pass volume with the content of `priv/grafana/provisioning` to the Grafana docker container.
+You can see, how it is done in this [docker-compose.yml](https://github.com/membraneframework/membrane_videoroom/blob/metrics-visualisation/docker-compose.yml)
+
+***NOTE 1*** If you will release a project with dependency to `membrane_rtc_engine_timescaledb`, content of `priv/grafana/provisioning` directory will be contained in the image 
```

Comment:
```suggestion
***NOTE 1*** If you release a project with dependency to `membrane_rtc_engine_timescaledb`, content of `priv/grafana/provisioning` directory will be contained in the image 
```

Reply:
BTW - wouldn't it be better to put those **NOTE**'s at the bottom of the section, after the environmental variables specification?